42-cellular-smoke-detector-retrofit/README.md

```diff
@@ -0,0 +1,184 @@
+
+Note that the instructions in this README require you to remove one of the smoke detectors in your home and install the hardware described here in its place. It's certainly possible to keep the smoke detector installed alongside the cellular monitoring system, but this is not covered in this document.
+
+## You Will Need
+
+* [Notecard](https://blues.io/products/notecard/)
+* [Notecarrier-A](https://shop.blues.io/collections/notecarrier/products/carr-al)
+* USB A to micro USB cable
+* [LiPo battery with JST cable](https://shop.blues.io/products/5-000-mah-lipo-battery)
+* [PC817 optocoupler](https://www.amazon.com/dp/B01JG8EJVW?psc=1&ref=ppx_yo2ov_dt_b_product_details)
+* Non-contact voltage tester
+* Digital multimeter with hook or clip probes
+* Sacrificial extension cord or similar cable for spare wire
+* Wire connectors
+* Resistors
+* Male-to-male jumper wires
+* Breadboard
+
+## Hardware Setup
+
+### Smoke Detector Wiring
+
+**WARNING: This application involves working with mains power. Getting zapped by mains power can be lethal. Do not proceed if you are unfamiliar with the risks involved.**
+
+Before you start building anything, identify the particular smoke detector in your home where you'll attach the new hardware. Remove the smoke detector from the ceiling/wall. The wiring in your home may vary, but you're likely to see the 3 wires of the smoke detector connected to the rest of the circuit with wire connectors:
+
+<p align="center">
+<img src="images/smoke_detector_wires.jpg"/>
+</p>
+
+Activate your voltage tester and bring it close to the black (hot) wire. You should hear a beep, indicating that the wire is live. Before you start changing the wiring, shut off power to this circuit by flipping off the corresponding circuit breaker in your home's breaker box. Use the voltage tester again to ensure power was shut off. You can now safely handle the wiring.
+
+To sense the voltage on the interconnect wire, we will need to install two additional wires: one that connects to the interconnect wire and another that connects to neutral. Find an extension cord or other sacrificial cable and cut it open to extract 2 lengths of wire:
+
+<p align="center">
+<img src="images/extension_wires.jpg"/>
+</p>
+
+Attach each wire to the interconnect wire connector and neutral wire connector, respectively:
```

Comment:
> Attach each wire

This is likely a very dumb question, but are you just shoving the new wires in there?

Reply:
Not quite haha. You unscrew the connector to expose all the wires, put the new one next to the old ones, and screw the connector back on. The connectors are threaded such that when you twist them (righty tighty) it will twist the wires together into one braid.